### PR TITLE
Margarita_Skorodnikova

### PR DESCRIPTION
**Pull request by Margareeta**

### CODE REVIEW

File: src/main/java/homework1/Main.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package homework1;
+
+public class Main {
+
+    public static String wordAndNumber (String word){
+
+        if (word.equals("ошибка")) {
+           return "\u001b[31m" + " ТРЕВОГА";
+        } else return word + ": " + word.length();
```
Comment (NikolaevArtem) [inline src/main/java/homework1/Main.java]: Doesn't comply with AC, as "letters" word in the end needed.

File: src/main/java/homework1/Main.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package homework1;
+
+public class Main {
+
+    public static String wordAndNumber (String word){
+
+        if (word.equals("ошибка")) {
+           return "\u001b[31m" + " ТРЕВОГА";
```
Comment (NikolaevArtem) [inline src/main/java/homework1/Main.java]: You forgot closing escape string. Also brackets after else. Also I don't see public static void main anywhere

File: src/main/java/homework1/Main.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package homework1;
+
+public class Main {
+
+    public static String wordAndNumber (String word){
```
Comment (NikolaevArtem) [inline src/main/java/homework1/Main.java]: Opt: Please fix style according to Code Conventions

File: src/main/java/homework1/Main.java (hunk 1 above)
Comment (Margareeta) [reply src/main/java/homework1/Main.java]: I'm sorry to say  I do not really understand what AC is. I added the word  letters to the end of this String, I hope I did that right.

File: src/main/java/homework1/Main.java [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+        } else if (!word.equals("ошибка")) {
+            return word + ": " + word.length() + " letters.";
+
+            //added closing String
+        } else return "";
+    }
+
+    // added psvm
+    public static void main(String[] args) {
+        System.out.println(wordAndNumberOfLetters(word));
```
Comment (NikolaevArtem) [inline src/main/java/homework1/Main.java]: Args are not passed to the method

File: src/main/java/homework1/Main.java [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+public class Main {
+    static String word;
+
+    public static String wordAndNumberOfLetters(String word) {
+
+
+        if (word.equals("ошибка")) {
+            return "\u001b[31m" + " ТРЕВОГА";
+
+        } else if (!word.equals("ошибка")) {
```
Comment (NikolaevArtem) [inline src/main/java/homework1/Main.java]: You dont need condition for this, it will come anyway. Just else {} is enough

**Comment by Margareeta** [comment]
Re-wrote the code

File: src/main/java/Homework1/Main.java [hunk 6]
```diff
@@ -0,0 +1,2 @@
+package Homework1;public class Main {
```
Comment (NikolaevArtem) [inline src/main/java/Homework1/Main.java]: Opt: If you don;t need this class, please delete

File: src/main/java/homework1/Main.java [hunk 7]
```diff
@@ -0,0 +1,17 @@
+package Homework1;
+
+public class Main {
+    public static void main(String[] args) {
+
+        for (String s : args) {
+            if (s.equals("error")) {
+                System.out.println("\u001b[31m" + "ALARM");
```
Comment (NikolaevArtem) [inline src/main/java/homework1/Main.java]: Opt: you forgot a closing escape string. If you don't use it, all your output until closing the application will be red. 
final String RED_COLOR = "\u001B[31m";
final String CLOSE_COLOR = "\u001B[0m";

**Comment by NikolaevArtem** [comment]
Seems forgotten

**Pull request closed without merging**